src/azure-cli/azure/cli/command_modules/rdbms/_params.py

```diff
@@ -718,6 +718,8 @@ def handle_migration_parameters(command_group, server_name_arg_type, migration_i
                                help='Request properties. Use double or no quotes to pass in filepath as argument.')
                     c.argument('migration_name', arg_type=migration_id_arg_type, options_list=['--migration-name'],
                                help='Name of the migration.')
+                    c.argument('migration_mode', arg_type=migration_id_arg_type, options_list=['--migration-mode'], required=False,
+                               help='Either offline or online(with CDC) migration. Deafult is offline', choices=['offline', 'online'], default='offline')
                 elif scope == "show":
```

Comment:
For `migration list` argument `--migration-filter` and `migration show` argument `--level`, we can also specify `choices=['Active', 'All']` and `default='All'`.